src/index.tsx

```diff
@@ -18,8 +18,9 @@ import 'react-toastify/dist/ReactToastify.css';
 import UserProvider from './hook/useAuth';
 const App = React.lazy(() => import('./App'));
 
+// https://atlp-bn.herokuapp.com/
 const httpLink = createHttpLink({
-  uri: 'https://atlp-bn.herokuapp.com/',
+  uri: 'http://localhost:4000/',
```

Comment:
Can we change this to use the environment variables instead of constantly changing everything.

If you face any issue setting up the environment variable usage, please ping me but we need such feature ASAP